source/includes/fundamentals/code-snippets/builders/AggregateSearchBuilderExample.java

```diff
@@ -0,0 +1,66 @@
+package fundamentals.builders;
```

Comment:
This class is to help future writers verify the syntax and behavior of this builder and potentially others.